landingzones/lz-machinelearning/networking.bicep

```diff
@@ -380,6 +406,26 @@ resource vnet 'Microsoft.Network/virtualNetworks@2021-02-01' = {
           privateEndpointNetworkPolicies: 'Disabled'
         }
       }
+      {
+        name: network.subnets.integration.name
```

Comment:
Since this is a delegated subnet and can only be used for App Service instances, this subnet should be renamed to "App Service"

Reply:
Resolved - renamed subnet as appService